pubspec.yaml

```diff
@@ -1,21 +1,23 @@
 name: flame_splash_screen
-description: Style your flame/flutter game with a beautiful splash screen. Simple to use but very customizable.
-version: 0.0.1
+description: Style your flame game with a beautiful splash screen with logo reveal. Simple to use but very customizable.
```

Comment:
What does "with logo reveal" mean? Not sure if it is wrong, but I don't understand it.

Reply:
Logo reveal is how they call these intros like [this one](https://www.youtube.com/watch?v=jmF0edcnw4w)